website/views.py

```diff
@@ -272,6 +272,11 @@ def _cache_middleware(request, *args, **kwargs):
     return _decorator
 
 
+@receiver(user_logged_out)
+def handle_user_logged_out(request, user, **kwargs):
+    cache.clear()
```

Comment:
Why do you clear the entire cache upon this event?

Reply:
yes sir, should i just remove the cache_key?

Reply:
I'm not sure what issue you fixed with that but it seem an overkill to me in any case as the **entire** cache is being invalidated upon any user signin/signup/logout event, right?

Reply:
yeah right sir , I will fix this up for sure